src/main/features/claim/views/check-and-send.njk

```diff
@@ -139,7 +139,7 @@
         {{ statementOfTruth(
             form,
             statementOfTruthType = SignatureType.CLAIM,
-            label = 'I believe that the facts stated in this claim are true.',
+            label = 'I believe that the facts stated in this claim are true.<p></p><p>I understand that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest belief in its truth.</p>',
```

Comment:
We shouldn't be using empty paragraphs to establish vspace. If the need for separate \<p>aragraphs is new, we should refactor or overload the `statementOfTruth()` method to accept an array and have it print out `<p>each line</p>`

Reply:
additional <p> tag was type ..... was testing something now I removed ... yes we have a new requirement to add paragraphs for statement of truth but it applicable  from the second item in an array if we implement an array ..... for now as its only one additional paragraph then I am adding as parameter .... in future any additional paragraphs comes we go for array approach